examples/playwright/token.config.js

```diff
@@ -0,0 +1,28 @@
+const { defineConfig, devices } = require('@playwright/test');
+
+/**
+ * @see https://playwright.dev/docs/test-configuration
+ */
+module.exports = defineConfig({
+  testDir: './tests',
+  reporter: [
+    ['line'],
+    ['buildkite-test-collector/playwright/reporter', { token: process.env.BUILDKITE_ANALYTICS_PLAYWRIGHT_TOKEN }]
```

Comment:
I'm not 100% how this works, but should this var match the one in the README?

Reply:
The token can be configured through ENV var using `BUILDKITE_ANALYTICS_TOKEN` key (`BUILDKITE_ANALYTICS_TOKEN=xyz npm test`) or via config file. When configuring the token via config file, it can be any Javascript expression (e.g. `token: 'xyz'`, `token: process.env.MY_TOKEN`). The key name doesn't matter when configuring the token via config file.